app/src/androidTest/java/ch/epfl/polychef/UserStorageTest.java

```diff
@@ -0,0 +1,249 @@
+package ch.epfl.polychef;
```

Comment:
This class is a duplicate of the refactored class in _user_, this one was changed thus you need to add your changes to the one in the _user_ package